src/index.js

```diff
@@ -3,48 +3,49 @@ import './css/reset.css';
 import './css/style.css';
 import j from './assets/jos.jpeg';
 import z from './assets/zieeco.jpeg';
-import render, { technologies } from './homeRender.js';
-import createPopup from './popup_comment.js';
-
-const htmls = document.getElementsByTagName('body');
-const popupContainer = document.querySelector('.pop--container');
-
-// const overLay = document.querySelector('.over--lay');
-
-technologies.forEach((project) => {
-  const popupCard = createPopup(project);
-  popupContainer.appendChild(popupCard);
-});
-
-const buttons = document.querySelectorAll('.comment');
-const closeButtons = document.querySelectorAll('#close');
-
-// console.log(buttons);
-
-// Display popup
-buttons.forEach((button) => {
-  const currentModal = document.getElementById(button.classList[1]);
-  // console.log(currentModal);
-  button.addEventListener('click', () => {
-    popupContainer.style.display = 'flex';
-    currentModal.classList.add('active');
-    htmls.style.overflow = 'hidden';
-  });
-});
-
-// remove popup
-closeButtons.forEach((closeBtn) => {
-  closeBtn.addEventListener('click', () => {
-    document.querySelector('.pop--container.active').classList.remove('active');
-    popupContainer.style.display = 'none';
-    htmls.style.overflow = 'auto';
-  });
-});
+// import renderTech from './homeRender.js';
+import getTechs from './set_api.js';
+// import createPopup from './popup_comment.js';
+
+// const htmls = document.getElementsByTagName('body');
+// const popupContainer = document.querySelector('.pop--container');
+
+// // const overLay = document.querySelector('.over--lay');
+
+// technologies.forEach((project) => {
+//   const popupCard = createPopup(project);
+//   popupContainer.appendChild(popupCard);
+// });
+
+// const buttons = document.querySelectorAll('.comment');
+// const closeButtons = document.querySelectorAll('#close');
+
+// // console.log(buttons);
+
+// // Display popup
+// buttons.forEach((button) => {
+//   const currentModal = document.getElementById(button.classList[1]);
+//   // console.log(currentModal);
```

Comment:
- [ optional ] Kindly, at the second stage, I will suggest you to remove these commented lines of code to make your code more readable